packages/gatsby-source-drupal/README.md

```diff
@@ -182,6 +182,12 @@ module.exports = {
 }
 ```
 
+_NOTES_:
+
+When using [includes](https://www.drupal.org/docs/8/modules/jsonapi/includes) in your JSON:API calls the included data will automatically become available to query, even if the link types are skipped using `disallowedLinkTypes`.
+
+This enables you to fetch only the data you need at build time, instead of all data of a certain entity type or bundle.
+
```

Comment:
It would be great to show an example of how this can be used in gatsby-config.js. The example in #21379 is fantastic for quickly showing how it can be used.